### ISSUE TITLE
Product export duplicate rows for product with html special chars in data

### ISSUE THREAD

**Issue by comdiler** [magento/magento2#7350]
### Preconditions
1. Tag 2.1.2

### Steps to reproduce

1. Open admin panel
2. Edit any product 
3. Add description that has quotes " or any other html special char
4. Save Product
5. Go to System - Export - Products
6. Export products

### Expected result
1. Product will be exported only once
<img width="672" alt="screen shot 2016-11-08 at 15 19 48" src="https://cloud.githubusercontent.com/assets/1975805/20102615/04ccb5d8-a5c7-11e6-9edd-84acd0358bca.png">

### Actual result
1. Product exported twice (can be more if have more than one store)
<img width="738" alt="screen shot 2016-11-08 at 15 19 55" src="https://cloud.githubusercontent.com/assets/1975805/20102626/0c03e358-a5c7-11e6-801d-642844ff43d9.png">

### Suggested fix 
Fix condition in if
<img width="1277" alt="screen shot 2016-11-08 at 15 23 02" src="https://cloud.githubusercontent.com/assets/1975805/20102689/47bb1394-a5c7-11e6-8820-b5783fc02139.png">

**Comment by xmav** [comment]
@comdiler  We've created internal ticket MAGETWO-63429 to address this issue.

**Comment by nntoan** [comment]
This was merged into 2.1.8

**Comment by magento-engcom-team** [comment]
@comdiler, thank you for your report.
The issue is already fixed in 2.2.0

**Comment by simonmaass** [comment]
I am having the exact same problem as described above! I am using magento 2.2.5!

The only difference that i can see between the products that exportet fine and the ones that have dublicates is the field "custom_options" - it looks like prducts with custom options get wrong export with dublicate products

![image](https://user-images.githubusercontent.com/32599710/43067975-5900890e-8e69-11e8-8a43-6cd52ae33d2d.png)

**Comment by dodiya-divyarajsinh** [comment]
I have same issue in Magento 2.2.5 @magento-engcom-team

**Comment by simonmaass** [comment]
why is this still closed?

**Comment by jeffreytu** [comment]
Same issue as @simonmaass in 2.2.6. Also my exports take 2 minutes now, whereas before it was only about 10s.

**Comment by simonmaass** [comment]
this issue doesnt seem fixed!

**Comment by Akki-8388** [comment]
getting same issue any one have solution

**Comment by simonmaass** [comment]
@magento-engcom-team any update here if this will get fixed?

**Comment by windymedia** [comment]
We are also seeing the same issue on 2.3.5 Composer install
